packages/react/src/components/cookieConsent/content/Content.tsx

```diff
@@ -45,12 +47,15 @@ export function Content(): React.ReactElement {
   }, [setFocusToTitle]);
```

Comment:
Not related to this PR, but browser console warns about unnecessary dependency currentLanguage on line 43:

>   Line 43:6:  React Hook useCallback has an unnecessary dependency: 'currentLanguage'. Either exclude it or remove the dependency array  react-hooks/exhaustive-deps

Reply:
Will be fixed in cookie-consent branch